src/sidebar/components/ShareDialog/ExportAnnotations.tsx

```diff
@@ -33,26 +34,31 @@ type ExportFormat = {
   /** Unique format identifier used also as file extension */
   value: 'json' | 'csv' | 'txt' | 'html';
   name: string;
+  description: string;
 };
 
 const exportFormats: ExportFormat[] = [
   {
     value: 'json',
     name: 'JSON',
+    description: 'For importing into another Hypothesis group or document',
```

Comment:
```suggestion
    description: 'For import into another Hypothesis group or document',
```

This saves a few characters and matches the other descriptions.